docs/operator.md

```diff
@@ -153,15 +90,79 @@ spec:
 EOF
 ```
 
-**NOTE Unsupported**
+## Configuring the Assisted Service Deployment
+
+### Via Subscription
+
+The operator subscription can be used to configure the images used in the
+assisted-service deployment and the installer + controller + agent images used by
+the assisted-service.
+
+``` bash
+cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: assisted-service-operator
+  namespace: assisted-installer
+spec:
+  channel: alpha
```

Comment:
I think `channel` is not required (or at least we are not using it in the dev-script script): 

https://github.com/openshift-metal3/dev-scripts/blob/6c831934eb5be222eef88f2587830279bf84741d/assisted_deployment.sh#L171-L185

Do you think it's worth keeping it as part of the documentation here? Anything that we should/could mention about the use of `channel` ?